src/components/Cart/OrderConfirmButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/react';
+import { useNavigate } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+
+import { isAllUnCheckedState } from '@recoil/cartItems/selectors';
+
+export default function OrderConfirmButton() {
+  const navigate = useNavigate();
+  const isAllUnChecked = useRecoilValue(isAllUnCheckedState);
+
+  const handleClickOrderConfirm = () => {
+    if (isAllUnChecked) return;
+
+    navigate('/confirm');
+  };
+
+  return (
+    <button
+      css={orderConfirmButton(isAllUnChecked)}
+      onClick={handleClickOrderConfirm}
+      disabled={isAllUnChecked}
+    >
+      주문확인
+    </button>
+  );
+}
+
+const orderConfirmButton = (isDisabled: boolean) => css`
+  width: 100%;
+  height: 64px;
+
+  background-color: ${isDisabled ? '#BEBEBE' : '#000'};
+
+  font-size: 16px;
+  font-weight: 700;
+  color: #fff;
+`;
```

Comment:
button 내부의 기존 disabled 속성을 사용해서 적용해줘도 괜찮을 것 같아요!
그렇다면, css에 별도의 prop을 전달해주지 않아도 될 것 같네용

```suggestion
const orderConfirmButton =  css`
  width: 100%;
  height: 64px;

  background-color: '#000';

  font-size: 16px;
  font-weight: 700;
  color: #fff;
  
  &:disabled {
    background-color: '#BEBEBE';
  }
`;
```

Reply:
이 부분은 혹시나 사용자 측에서 악의적으로 `disabled` 속성을 조작했을 때에 대비해서 추가적으로 속성을 적용해주었습니다!

Reply:
오호... 그런 이유가 있었군요!! 섬세함 🫡🫡